DependencyInjection/Compiler/AddAuditEntityCompilerPass.php

```diff
@@ -14,10 +14,6 @@
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
-/*
- *
- * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
```

Comment:
Thought this is redundant because of the existing header of every single php file https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/master/DependencyInjection/Compiler/AddAuditEntityCompilerPass.php#L3-L10